src/pages/dashboard/index.tsx

```diff
@@ -47,9 +48,18 @@ const OutputSection: React.FC<OutputSectionProps> = ({ shortUrl, handleCopyUrl }
             value={shortUrl}
             placeholder="Copy the URL"
         />
+        <a
+            type="button"
+            className="bg-gray-200  px-2 py-4 hover:bg-gray-400"
+            href={shortUrl}
+            target="_blank"
+            rel="noopener noreferrer"
+        >
+            <ShareIcon />
+        </a>
```

Comment:
Generally have seen Link tag being used in Next Js inplace of anchor tag

Reply:
Done